src/Compilers/CSharp/Portable/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -154,6 +154,13 @@ internal static RefKind GetRefKind(this TypeSyntax syntax)
             return refKind;
         }
 
+        internal static TypeSyntax SkipRef(this TypeSyntax syntax)
```

Comment:
i think someone asked to add overloads that return only what is needed in past code reviews. 
having an "out" parameter that is unused by the caller was causing cognitive overload. The implementation is also much simpler.

---
In reply to: [113553925](https://github.com/dotnet/roslyn/pull/18715#discussion_r113553925) [](ancestors = 113553925)